src/OrbitGl/TranslationStack.h

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_GL_TRANSFORMATION_STACK_H_
+#define ORBIT_GL_TRANSFORMATION_STACK_H_
```

Comment:
The include guard says transformation stack while the class is called translation stack